### PR TITLE
Fix failing tests where we use no longer supported pattern expressions

### PR DESCRIPTION
**Pull request by LinneaAndersson**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ScalarFunctionsTest.scala [hunk 1]
```diff
@@ -266,13 +266,13 @@ class ScalarFunctionsTest extends DocumentingTest {
       }
     }
     section("size() applied to pattern expression", "functions-size-of-pattern-expression") {
```
Comment (hedengran) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ScalarFunctionsTest.scala]: You forgot to update the title.

**Review by hedengran — COMMENTED** [review]
Nice! Just update the title and this should be ready for merge.

**Review by hedengran — APPROVED** [review]
👍

**Review by renetapopova — APPROVED** [review]
LGTM, thanks @LinneaAndersson!

**Pull request merged**